.github/workflows/dist.yml

```diff
@@ -32,7 +31,7 @@ jobs:
             app_name: lpc55xpresso
             app_toml: app/lpc55xpresso/app.toml
             target: thumbv8m.main-none-eabihf
-            image: [a, b]
+            image: "a, b"
```

Comment:
i am interested in this change, I'm not sure why it's being done

Reply:
Reusable workflows can only handle inputs of boolean, number or string https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callinputsinput_idtype so I converted this to a comma separated string. It turns out we were already doing this in the build anyway.

Reply:
Ah, interesting, thank you!